app/models/recent_activity.rb

```diff
@@ -69,6 +69,12 @@ def self.user_activity(user, reject_keys=[])
   # All activities that are public and should be visible for a user
   # * +user+ - the user which activities will be returned
   def self.user_public_activity user
```

Comment:
Yeah, you're right.
But still, this `select` ruins the purpose of pagination.
This would work, even though it looks terrible:

```ruby
user_activity(user, ["space.decline"])
  .where("activities.parameters REGEXP '#{{ user_id: user.id }.to_yaml}'")
```

But maybe the user should always be the owner, so it wouldn't need to be in the parameters. We could even extend this to "nothing that will be used in queries should be in the parameters".

We could have more columns in RecentActivity, to have something like `trackable`, `owner` and `container`. One example would be a "post.create", that would have `trackable` as the post itself, `owner` as the user that created it, and `container` as the space were it was created.

Any other idea?